### PR TITLE
Modifies deploy files to configure certbot

### PR DESCRIPTION
**Pull request by mariestaver**
I based these file changes, additions & deletions on this similar commit in /bocoup/api: https://github.com/bocoup/api/commit/15cc0ee38e7acece78000f9484fe323cc62868bf#diff-dd915a9c2a2df48683162f3806364e989b904a9a8a196bdb1a15398ba1805d5dR68

### CODE REVIEW

File: deploy/ansible/host_vars/vagrant [hunk 1]
```diff
@@ -18,9 +18,9 @@ hostname: ansible-vagrant
...
 # Should the nginx server use HTTPS instead of HTTP?
-ssl: false
+# ssl: false
 
 # If ssl is enabled, these cert/key files will be used by nginx. Self-signed
 # cert/key files will be auto-generated if they don't already exist.
-ssl_cert_path: /etc/ssl/server.cert
-ssl_key_path: /etc/ssl/server.key
+# ssl_cert_path: /etc/ssl/server.cert
+# ssl_key_path: /etc/ssl/server.key
```
Comment (jugglinmike) [inline deploy/ansible/host_vars/vagrant:26]: We can remove all this

File: deploy/ansible/roles/certbot/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- name: Install Certbot
+  apt:
+    name: python3-certbot-nginx
+    state: present
+
+- name: Run Certbot
+  command: certbot -m infrastructure+api.bocoup.com@bocoup.com --agree-tos -n --domains {{api_fqdn}} --nginx
```
Comment (jugglinmike) [inline deploy/ansible/roles/certbot/tasks/main.yml:7]:
```suggestion
  command: certbot -m {{letsencrypt_email}} --agree-tos -n --domains {{api_fqdn}} --nginx
```

**Review by jugglinmike — CHANGES_REQUESTED** [review]

File: deploy/ansible/host_vars/vagrant (hunk 1 above)
Comment (mariestaver) [reply deploy/ansible/host_vars/vagrant]: removed in most recent commit -- thanks!

**Comment by mariestaver** [comment]
Hi @jugglinmike -- I made the changes we discussed (and a few extra commits to fix minor formatting) -- I believe this can be merged now.  Let me know if you agree, and thank you again for your help with this process!

File: deploy/ansible/roles/certbot/tasks/main.yml [hunk 3]
```diff
@@ -11,4 +11,5 @@
     name: Certbot automatic renewal.
     job: "/usr/bin/certbot renew --quiet --no-self-upgrade && service nginx restart"
     minute: 0
-    hour: 23
\ No newline at end of file
+    hour: 23
+
```
Comment (jugglinmike) [inline deploy/ansible/roles/certbot/tasks/main.yml:15]: This file still lacks a trailing newline

File: deploy/ansible/roles/certbot/tasks/main.yml (hunk 2 above)
Comment (jugglinmike) [reply deploy/ansible/roles/certbot/tasks/main.yml:7]: It looks like this suggestion got lost in the shuffle. Mind making the replacement?

Comment (mariestaver) [reply deploy/ansible/roles/certbot/tasks/main.yml]: Sorry about that!  Thank you for the catch -- New commit with this change (and the newline): https://github.com/bocoup/people/pull/55/commits/288679cd883dd6b6666449b848416f83c00effcd

File: deploy/ansible/roles/certbot/tasks/main.yml (hunk 3 above)
Comment (mariestaver) [reply deploy/ansible/roles/certbot/tasks/main.yml:16]: Gah, I could have sword I caught all those!  Thank you :)  Commit with fix: https://github.com/bocoup/people/pull/55/commits/288679cd883dd6b6666449b848416f83c00effcd

**Review by jugglinmike — APPROVED** [review]

**Pull request merged**